releng/jenkins/release-simrel-update/Jenkinsfile

```diff
@@ -80,7 +76,10 @@ pipeline {
                 XTEXT_VERSION_LABEL=$(echo $XTEXT_VERSION | sed "s|\\(.*\\)\\.v.*|\\1.$RELEASE_TYPE|");;
             esac
             git diff
-            git commit --signoff -m "Xtext $XTEXT_VERSION_LABEL contribution to SimRel $SIMREL_LABEL"
+            git --version
+            git commit --signoff -m "Xtext $XTEXT_VERSION_LABEL contribution to SimRel $SIMREL_LABEL
+            
+Change-Id: I0000000000000000000000000000000000000000"
```

Comment:
is this really the problem?

Reply:
No, this slipped in accidentially. Only agent and grep command should change. Sorry for that, will revert these lines now.

Reply:
@cdietrich updated